draft-ietf-cats-framework.md

```diff
@@ -159,7 +153,15 @@ Service:
   : The same service can be provided in many locations; each of them constitutes a service instance.
 
 Computing Service:
-  : An offering is made available by a provider by orchestrating a set of computing resources (without networking resources).
+  : An offering that is made available by a provider by orchestrating a set of computing resources to process application content.
+
+  : Some differences and similarities exist between service and computing service:
```

Comment:
"computing service" is still a service! Also, why are we adding this diff here? How this is linked to "application" mentioned right before?

Reply:
change it to service? or just delete it. As long as the definition is the same in use case and framework draft, I am ok.